notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    bucket = storage.Client(project=PROJECT_ID).bucket(BUCKET_NAME.replace("gs://", ""))
</pre><p>Why use gsutil earlier and storage client here? Use gsutil if possible to be consistent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>